packages/block-library/src/template-part/edit/inner-blocks.js

```diff
@@ -127,11 +127,14 @@ export default function TemplatePartInnerBlocks( {
 	const { canViewTemplatePart, canEditTemplatePart } = useSelect(
 		( select ) => {
 			return {
-				canViewTemplatePart:
-					select( coreStore ).canUser( 'read', 'templates' ) ?? false,
-				canEditTemplatePart:
-					select( coreStore ).canUser( 'create', 'templates' ) ??
-					false,
+				canViewTemplatePart: !! select( coreStore ).canUser( 'read', {
+					kind: 'postType',
+					name: 'wp_template',
```

Comment:
Similar to what @ellatrix suggested about the template part post type, we could use `TEMPLATE_POST_TYPE` for these.

Reply:
I've updated the part when these constants were available. They're package-specific. Most entity-related blocks hardcode their entity names.

Are we worried that these names might change?

Also, I don't mind defining a couple of constants at the file level, but for the initial run, I tried to change only the selector syntax and leave the rest of the code alone.

Reply:
Definitely not a blocker for the PR 👍